packages/localization/src/DateFormatter.js

```diff
@@ -46,7 +47,7 @@ function pad(number) {
  * formatter. Every function is localized and you should never call formatting
  * on the raw Date object.
  */
-function DateFormatter(rawDate: Date = new Date()) {
+function DateFormatter(rawDate: Date = DateUtils.getUTCNow()) {
```

Comment:
I still believe this method should not accept the `Date` type... :)

Reply:
Yes, maybe for a later refactor.